src/proofPoints.ts

```diff
@@ -658,6 +659,7 @@ class ProofPointRegistry {
 
             return web3.utils.toChecksumAddress(didDocument.publicKey[0].ethereumAddress);
         } catch(e) {
+          console.log(e);
```

Comment:
Needed?

Could we add the same eslint checks to this project as we have in others? (separate PR)